source/opt/ir_context.h

```diff
@@ -187,6 +187,14 @@ class IRContext {
   inline IteratorRange<Module::inst_iterator> debugs3();
   inline IteratorRange<Module::const_inst_iterator> debugs3() const;
 
+  // Iterators for debug info instructions (excluding OpLine & OpNoLine)
+  // contained in this module.  These are OpExtInst for OpenCL.DebugInfo.100
+  // or DebugInfo extension placed between section 9 and 10.
+  inline Module::inst_iterator debuginfo_begin();
```

Comment:
The name is confusing because there are so many kinds of debug info.
Please rename this to ext_inst_debuginfo or ext_inst_debug_info